script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1_2.txt

```diff
@@ -36,7 +36,7 @@ Although getting taken in by food is something a cat would do."
 <Page12>
 Her smile is so friendly that I couldn't help but joke a little.
 <Page13>
-The green ribbon wearing Senpai spoke,
+The green ribbon-wearing Senpai spoke,
```

Comment:
I think with the hyphen here this evaluates to The (green (ribbon-wearing senpai)), which is to say a green senpai wearing a ribbon, rather than The (((green ribbon) wearing) senpai), which would be a senpai wearing a green ribbon